disk_usage/dir_scanner.py

```diff
@@ -0,0 +1,80 @@
+import os
+from functools import lru_cache
+from typing import Callable, List
```

Comment:
```suggestion
from typing import Callable
```
используем встроенный `list`